README.md

```diff
@@ -177,6 +177,27 @@ Version: 3
 Status: succeeded
 ```
 
+**~deploy_app**
+
+If you would like to deploy your application, you can use this command:
+
+```
+you: ~deploy_app app_name link_to_tarball_of_code version_to_release
```

Comment:
I'm not sure this is the command we'd want to use, as we don't currently tag crates.io releases. The command I'd use is:

```
~deploy_app crates-io rust-lang/crates.io COMMIT_REF
```

...where `COMMIT_REF` is either a branch name or a (partial?) commit sha.

To implement this, there is a GitHub API [to resolve an arbitrary git ref as a commit](https://api.github.com/repos/rust-lang/rust/commits/master) (the nice thing about that API is, you can use arbitrary git refs, even things like `master^1` to get the parent of the commit on master). Once you get the full SHA you can build the tarball URL:

```
https://codeload.github.com/{org}/{repo}/tar.gz/{sha}
```

Reply:
Done!